packages/shared/src/components/cards/AdList.tsx

```diff
@@ -21,12 +20,12 @@ export const AdList = forwardRef(function AdList(
   }, []);
 
   return (
-    <ListCard {...props} className={classNames(className, styles.ad)} ref={ref}>
+    <ListCard {...props} className={classNames(className)} ref={ref}>
```

Comment:
No longer needed as `className` is part of the `{...props}`

```suggestion
    <ListCard {...props} ref={ref}>
```